### ISSUE TITLE
Luminosity ratio of the place holder text and disabled text input is less than 4.5:1.

### ISSUE THREAD

**Issue by aubreyquinn** [microsoft/fluentui#19499]
### Environment Information

- **Package version(s)**: 0.54.0
- **Browser and OS versions**: Edge, Windows 10

### Describe the issue:

Luminosity contrast ratio of the disabled and Inverted Input box text is less than required contrast ratio 4.5:1.

text: rgb(200, 198, 196) #C8C6C4
color: rgb(243, 242, 241) #F3F2F1
Contrast is 1.52

### Please provide a reproduction of the issue in a codepen:

Here is my codepen: https://codesandbox.io/s/fluent-ui-example-forked-nrz89?file=/example.js

#### Actual behavior:

When the Input component is inverted and disabled, it is not longer meeting the a11y requirements.

#### Expected behavior:

4.5:1 is the expected ratio.

### Documentation describing expected behavior

Regarding colors, the standard defines two levels of contrast ratio: AA (minimum contrast) and AAA (enhanced contrast).
The level AA requires a contrast ratio of at least 4.5:1 for normal text and 3:1 for large text (at least 18pt) or bold text.
The level AAA requires a contrast ratio of at least 7:1 for normal text and 4.5:1 for large text or bold text.

**Comment by gouttierre** [comment]
@aubreyquinn - thanks for filing this issue with us. 

@jurokapsiar -  Would your team be able to confirm if this is a regression, or if this behavior is an issue?

**Comment by aubreyquinn** [comment]
@jurokapsiar yes, it's still an issue for v 0.54

**Comment by jurokapsiar** [comment]
Hi, thank you for reporting. We discussed this with designers - if a component is inactive it doesn't have to meet 4:5:1 - https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html

> Incidental
> Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.
